pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -255,76 +355,83 @@ def _load_connections_file(self):
                        format(B08_DEFAULT_CONNECTIONS_FILE,
                               DEFAULT_CONNECTIONS_FILE))
 
+        # load the existing file
         if os.path.isfile(self._connections_file):
             with self._open_file(self._connections_file, 'r') as _fp:
```

Comment:
Maybe add handling of `IOError` around the `with self._open_file(...`, turning this also into `ConnectionsFileLoadError`.